Description: Remove unused, unpackaged yet deps
Last-Update: 2023-10-31
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -48,5 +48,0 @@
-[dependencies.cidr-02]
-version = "0.2"
-optional = true
-package = "cidr"
-
@@ -80,12 +75,6 @@
 optional = true
 package = "serde_json"
 
-[dependencies.smol_str-01]
-version = "0.1.23"
-optional = true
-default-features = false
-package = "smol_str"
-
 [dependencies.time-03]
 version = "0.3"
 optional = true
@@ -102,1 +91,0 @@
-with-cidr-0_2 = ["cidr-02"]
@@ -110,1 +110,0 @@
-with-smol_str-01 = ["smol_str-01"]
